src/main/java/net/minestom/server/utils/time/Tick.java

```diff
@@ -0,0 +1,45 @@
+package net.minestom.server.utils.time;
+
+import net.minestom.server.MinecraftServer;
+
+import java.time.Duration;
+import java.time.temporal.Temporal;
+import java.time.temporal.TemporalUnit;
+
+public class Tick implements TemporalUnit {
```

Comment:
I prefer my implementation of this class (in #205). Feel free to nab it to replace this if you want.

Reply:
I just checked that pr after i pushed this, but i'll include it too or if it gets merged before this i'll rebase.

Reply:
Well I can just copy-paste it and i'll have less things to do if i have to rebase... Soon i'll push it and resolve this